tests/queries/0_stateless/01921_test_progress_bar.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+import os
+import sys
+import signal
+
+CURDIR = os.path.dirname(os.path.realpath(__file__))
+sys.path.insert(0, os.path.join(CURDIR, 'helpers'))
+
+from client import client, prompt, end_of_block
+
+log = None
+# uncomment the line below for debugging
+#log=sys.stdout
+
+with client(name='client1>', log=log) as client1, client(name='client2>', log=log) as client2:
```

Comment:
`client2` is never used (looks like a copy-paste from `*live_view*` tests)

Btw, it's possible to write such tests without wrappers on python, for example see `01520_client_print_query_id`, `01179_insert_values_semicolon` and other `*.expect` tests. But tests on python are ok, as to me.